docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -22,29 +22,50 @@ for more information on these two file transfer sources.
 
 By default, Docker for Mac allocates fewer resources (CPU, Memory, Swap)
 to containers compared to what is available on your host machine. You
 can change what it allocates using the Docker for Mac GUI under
 ``Preferences > Advanced`` as described
 `here <https://docs.docker.com/docker-for-mac/#advanced>`__.
 
-* The default allocation can cause workflows or tools to fail without informing the user with a memory or resource related error message. If you find that your workflow or tool is behaving differently on a Mac compared to a similarly resourced Ubuntu environment, you can try increasing the resources allocated to Docker on the Mac to resolve the discrepancy.
+The default allocation can cause workflows or tools to fail without informing the user with a memory or resource related error message. If you find that your workflow or tool is behaving differently on a Mac compared to a similarly resourced Ubuntu environment, you can try increasing the resources allocated to Docker on the Mac to resolve the discrepancy. If you are using WDL, see also our notes on :doc:`local Cromwell configuration files`</advanced-topics/dockstore-cli/local-cromwell-config>`.
+
+Why am I getting "Mount denied" errors when launching workflows on Mac?
+-----------------------------------------------------------------------
+Docker behaves a bit differently on a `Mac <https://docs.docker.com/docker-for-mac/osxfs/#/namespaces>`__ than on a typical Ubuntu machine. On a Mac, by default, the only shared volumes Docker can access are /Users, /Volumes, /tmp, and /private. This is problematic because many workflow executors, such as `cwltool`, uses your TMPDIR (the :ref:`dict environment variable`) to setup volumes with Docker. Since `$TMDIR` on a Mac can default to a subdirectory of /var, which Docker usually cannot access, you may run into an error. When using cwltool, the error will look something like this:
+
+```
+docker: Error response from daemon: Mounts denied: cker-for-mac/osxfs/#namespaces for more info.
+.
+l_vvgsbw0000gn/T/tmptNfjZl/./empty_copy.txt
+is not shared from OS X and is not known to Docker.
+You can configure shared paths from Docker -> Preferences... -> File Sharing.
+See https://docs.docker.com/do.
+ERRO[0000] error getting events from daemon: net/http: request canceled 
+```
+
+This issue can happen when using cwltool or Cromwell, with or without the Dockstore CLI. There are two possible ways to fix this. 
+
+Option A: Overwrite $TMPDIR
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can set the $TMPDIR :ref:`dict environment variable` to be under one of the shared volumes that Docker for Mac can access by default. You can do this by doing something similar to the following:
```

Comment:
I've never done this, but I assume that you need to do this before you start Docker? If so, can you add the clarification?

Reply:
I don't know for sure but I will add that in.